packages/core/examples/buttonsExample.tsx

```diff
@@ -19,6 +19,7 @@ export interface IButtonsExampleState {
     loading?: boolean;
     large?: boolean;
     minimal?: boolean;
+    placeholder?: boolean;
```

Comment:
I've been thinking about the naming and I'm torn between `placeholder` and `muted`. We've been using `muted` already as a modifier for text, and it seems a bit more generic than `placeholder`. Would love to hear your thoughts! Also @cmslewis what do you think?

Reply:
No strong opinion, but now that you mention it, `placeholder` elsewhere means a `string`:

![image](https://user-images.githubusercontent.com/443450/30759124-d7e402fc-9f89-11e7-9955-84d0123b836e.png)

So maybe `muted` would be clearer?

Reply:
Very good point. What do you think about changing to `.pt-muted` @brieb? I think I buy it

Reply:
yeah def cannot use `placeholder` cuz that's an HTML prop that accepts a string.

we should reuse `.pt-text-muted`, if anything.

Reply:
Named it `placeholder` to more clearly associate it with `::placeholder`. 
Muted works too, though.